trin-core/src/portalnet/types/messages.rs

```diff
@@ -491,6 +491,13 @@ pub struct Accept {
     pub content_keys: BitList<typenum::U8>,
 }
 
+impl Into<Value> for Accept {
+    fn into(self) -> Value {
+        // TODO: self.content_keys returns a number of set bits to the json-rpc client, probably we would like to return a list of 0 and 1 bits
+        serde_json::json!({ "connection_id": format!("{:?}", self.connection_id.to_be()) , "content_keys": self.content_keys.num_set_bits()})
```

Comment:
I think we can go ahead and return the hex-string for `content_keys`, which seems to be the type in the initial JSON-RPC spec.